Dockerfile

```diff
@@ -15,6 +15,8 @@
 # Reproducible builder image
 FROM openshift/origin-release:golang-1.10 as builder
 
+RUN yum install -y libvirt-devel
```

Comment:
Do not add this as `libvirt-devel` is already in `openshift/origin-release:golang-1.10` image

Reply:
Looks like this was already fixed in master. This bit is gone after a rebase.